woocommerce/google-listings-and-ads
# Get started page UX improvements: add benefits card

### Changes proposed in this Pull Request:

Part of #1392, this PR adds the benefits card to the new get started page.

Design for desktop: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121106
Design for mobile: eQ9O4m2flzrcAiDMXkOW0m-fi-7476%3A122821

### Screenshots:

#### Desktop

<img width="1288" alt="Screenshot 2022-04-21 at 15 00 21" src="https://user-images.githubusercontent.com/914406/164393372-cbf74076-88bd-4007-94d7-9b3e74a12871.png">

#### Mobile

<img width="1281" alt="Screenshot 2022-04-21 at 15 00 32" src="https://user-images.githubusercontent.com/914406/164393395-499a20c2-3082-46e9-97e9-652ff51beda3.png">

### Detailed test instructions:

1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the second card of the get started page, the UI should align with the design.


### Changelog entry

> Update - Add the benefits card for UX improvements on get started page.
